sklearn/ensemble/_voting.py

```diff
@@ -144,6 +146,14 @@ def _sk_visual_block_(self):
     def _more_tags(self):
         return {"preserves_dtype": []}
 
+    def get_feature_names_out(self, input_features=None):
+        _check_feature_names_in(self, input_features)
+        class_name = self.__class__.__name__.lower()
+        return np.asarray(
+            [f"{class_name}{i}" for i in range(len(self.estimators_))],
+            dtype=object,
+        )
+
```

Comment:
same comment here. If you don't want to reuse the `_ClassNamePrefixFeaturesOutMixin` mixin-class, you will need to set a valid docstring for this public method.

I think using the mixing class is probably best here as well.